cogs/source/source.py

```diff
@@ -0,0 +1,146 @@
+from redbot.core import commands
+import tracemoepy
+from tracemoepy.errors import TooManyRequests
+from tracemoepy.errors import EntityTooLarge
+from tracemoepy.errors import ServerError
+from tracemoepy.errors import EmptyImage
+import math
```

Comment:
Are we using this import?

Reply:
I thought it was but turns it wasn't needed